src/lib/pgp-key.h

```diff
@@ -271,7 +271,7 @@ pgp_subsig_t *pgp_key_get_subsig(const pgp_key_t *, size_t);
 
 void pgp_subsig_free(pgp_subsig_t *subsig);
 
-pgp_rawpacket_t *pgp_key_add_rawpacket(pgp_key_t *, void *, size_t, pgp_content_enum);
+pgp_rawpacket_t *pgp_key_add_rawpacket(pgp_key_t *, void *, size_t, pgp_pkt_type_t);
```

Comment:
why some function declarations have parameter names and some don't? Don't we want to make things uniform?

Reply:
@rrrooommmaaa that's how historically it was done in NetPGP, and some new code just follows this rule. For sure code would be more readable with parameter names. However, there is some discussion regarding this:

https://wiki.sei.cmu.edu/confluence/display/c/API08-C.+Avoid+parameter+names+in+a+function+prototype